rpc/src/rpc_trait.rs

```diff
@@ -19,101 +19,100 @@
 use crate::{error::RpcError, rpc_types::*};
 use snarkos_metrics::snapshots::NodeStats;
 
-use jsonrpc_derive::rpc;
-
 use std::net::SocketAddr;
 
 /// Definition of public RPC endpoints.
-#[rpc]
+#[async_trait::async_trait]
 pub trait RpcFunctions {
     // todo: readd in Rust 1.54
     // #[cfg_attr(nightly, doc(include = "../documentation/public_endpoints/getblock.md"))]
-    #[rpc(name = "getblock")]
-    fn get_block(&self, block_hash_string: String) -> Result<BlockInfo, RpcError>;
+    // #[rpc(name = "getblock")]
```

Comment:
(applies to all instances of this attribute being commented out) can we remove these attributes or is this some temporary measure? in case of the latter, a comment is needed